### PR TITLE
Deal with 1 hit case in `voxelize_hits`

### PR DESCRIPTION
**Pull request by paolafer**
A test has been implemented to make the voxelize_hits function of paolina able to deal with 1-hit events.

### CODE REVIEW

File: invisible_cities/reco/paolina_functions_test.py [hunk 1]
```diff
@@ -201,3 +202,12 @@ def test_blobs(track_extrema, radius, expected):
...
     assert blob_energies(track, radius) == (Ea, Eb)
+
+def test_one_hit_case():
+    hits = []
+    one_hit = BHit(1, 1, 1, 100)
+    hits.append(one_hit)
+    vox_size = np.array([10,10,10],dtype=np.int16)
+    voxelize_hits(hits, vox_size)
+
+    return len(voxelize_hits(hits))
```
Comment (jacg) [inline invisible_cities/reco/paolina_functions_test.py]: Tests should

+ not return anythyng
+ contain an assertion

**Comment by jacg** [comment]
This addresses #328.

**Comment by jacg** [comment]
Good, that's it.

I am struggling to find the time to finish #324 before I go offline for a few weeks, so if anyone has the time and inclination to fix this, feel free to go ahead.

**Comment by jacg** [comment]
Hmm, you've committed this on your `master` branch. That's bad. It's going to complicate working on it.

Please get in the habit of never committing onto `master`. Things only should get into `master` by merging.

This should have been done on some new branch with a name like `voxelize-single-hit`.

**Comment by jacg** [comment]
This PR is now ready for review.

The solution to the problem is contained entirely in the last commit. The other commits merely take the opportunity to clean up some minor issues in the relevant module.

**Comment by jjgomezcadenas** [comment]
Please, @gonzaponte, review. Please @paolafer, take into account the Master's comments.

**Comment by paolafer** [comment]
Copy that!

**Comment by gonzaponte** [comment]
The function `shortest_paths` seems a bit superfluous, but:
- Makes its functionality more obvious.
- We have a docstring.
- We have type annotations.

All these being very helpful for the non-expert reader. I am very happy with this.

**Review by gonzaponte — APPROVED** [review]
This PR includes a solution to a special case in the input of `voxelize_hits`. A test has been added and the code cleaned up a bit. Approved.

**Pull request merged**